data/stop_deviations.yml

```diff
@@ -0,0 +1,45 @@
+# Sometimes stop-level data is missing when in fact route was surveyed
+# and stop-level data was collected for the rest of the route.
+#
+# This deviation could be for 2 reasons:
+# 1) The observer(s) were unable to access the stop for safety or other reasons.
+#    In this case, we do not know the bird count.
+# 2) The stop was in fact observed, 
+#    but no birds were seen.
+#    The eBird download does not contain the empty checklist.
+#    In this case, we do know the count of all species is 0.
+#
+# This file tracks these cases.
+
+- year : 2023
+  county : Chatham
+  route : 3
+  stops_nobirds : [12]
+  stops_skipped : []
+
+- year : 2024
+  county : Chatham
+  route : 3
+  stops_nobirds : [12]
+  stops_skipped : []
+
+- year : 2021
+  county : Orange
+  route : 8
+  # https://ebird.org/checklist/S88387941
+  stops_nobirds : [1]
+  stops_skipped : []
+
+- year : 2023
+  county : Orange
+  route : 8
+  # https://ebird.org/checklist/S138614693
+  stops_nobirds : [1]
+  stops_skipped : []
+
+- year : 2022
+  county : Orange
+  route : 2
+  stops_nobirds : []
+  # reasons?
+  stops_skipped : [2, 3, 4]
```

Comment:
@IJBG - do you know why these are missing?

Reply:
The stops were missed for construction blocking the road [2,3,4]